### DISCUSSION TITLE
Use cached image when image not retrevied

### DISCUSSION THREAD

**Discussion by orzechszek** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#517] [Ideas]
Hi,

I've noticed, when image is not retrevied, error with wrong calibration appears, so choosing zone is not possible and vacuum is not usable in this case.
So I have a question - is it possible to add functionality something like 'use_cached_image', when camera is configured with 'store_map_image' to keep mentioned functionality and use already downloaded view? It will prevent of blocking zone functionality.

**Comment by PiotrMachowski** [comment]
Yes, I'm in the process of implementation of such approach

**Reply by orzechszek** [reply]
Thanks for info !! :) . When this feature can be expected? There is also another issue and because of that image doesn't appear at all.

**Reply by PiotrMachowski** [reply]
Yeah, that requires me to rewrite it again, so it might take some time
